test/responses/G21/json_export.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
Pretty sure this will break the tests as the whole file got refactored/formatted. Please only add the new lines where they occur (you can use the `bimmerconnected fingerprint` of your own vehicle as reference).